packages/suite/src/reducers/suite/desktopUpdateReducer.ts

```diff
@@ -0,0 +1,51 @@
+import produce from 'immer';
+import { DESKTOP_UPDATE } from '@suite-actions/constants';
+import { Action } from '@suite-types';
+import { UpdateInfo, UpdateProgress, UpdateWindow } from '@suite-types/desktop';
+
+export interface State {
+    state?: 'checking' | 'available' | 'not-available' | 'downloading' | 'ready';
+    skip?: string;
```

Comment:
micro nitpick: maybe comments here what these variables mean? Might improve code readability. I am reading this for the first time and asking myself what may optional `skip` string mean.